cmd/main.go

```diff
@@ -188,3 +191,20 @@ func getProjectVersion() (bool, string) {
 	}
 	return true, projectInfo.Version
 }
+
+func printV1DeprecationWarning() {
+	fmt.Printf(`=============== DEPRECATION WARNING ===============
```

Comment:
If this looks too aggressive, let me know how to rephrase/reformat it.

Reply:
We can try making it more concise. How about:
```
warning: [deprecation] v1 projects are deprecated and will not be supported beyond Feb 1, 2020. Refer to https://book.kubebuilder.io/migration/guide.html for more details.
```

Reply:
Thanks for the suggestion! Done